src/components/AnnouncementInfo/AnnouncementInfo.tsx

```diff
@@ -47,6 +49,24 @@ const AnnouncementInfo = ({
               {props.ctaText}
             </LinkTo>
           )}
+
+          {props.link.discriminant === 'document' && (
+            // We need to use the default Next Link component
+            // with legacyBehavior=false, so we can pass in an onClick
+            // that will open PDFs in the browser
+
+            <Link
+              legacyBehavior={false}
+              onClick={(e) => {
+                e.preventDefault()
+                handleOpenPdfLink(props.link.value.data?.file?.url)
```

Comment:
Currently, the pdf functionality works as expected! I uploaded a rather large file, and it allowed me to view it in the browser without downloading. However, I can't get any other file types to download. I've tried uploading rtf, docx, and txt files, and when I click the CTA nothing happens. No errors anywhere, just the button remains focused. I think it's because we're only calling `handleOpenPdfLink` here, and if the file extension is not pdf, we return.